src/surfaces/actionBar/actionBar.jsx

```diff
@@ -180,6 +189,25 @@ class ActionBar extends React.PureComponent {
         });
     }
 
+    setMobileSearchAutoFocus() {
+        const {
+            columns,
+        } = this.props;
+
+        let hasEventBeenCalled = false;
+
+        forEach(columns, (c) => {
+            if (c.list) {
+                forEach(c.list, (l) => {
+                    if (!hasEventBeenCalled && l.iconSearch?.autoFocus) {
+                        this.onMobileSearchIconToggle();
+                        hasEventBeenCalled = true;
```

Comment:
Should we add `break;` here as well to short-circuit any remaining loop iterations?

Reply:
You can use `break;` in a forEach? Here's what I get from my editor...

![image](https://user-images.githubusercontent.com/1100790/134597716-57d0033f-f0ae-4a12-8fde-8bca6a875905.png)

Reply:
Stinkin' JavaScript!  You can totally do that in C#!
Ah well ... don't mind me!  Nothing to see here! =)

Reply:
You know what is funny, I thought while making this change, "I wish you could break out of these loops easily, but I cannot." This is probably obvious as to why i implemented the `hasEventBeenCalled` boolean. Then you commented, about `break;` and I thought, "oh, man, you can, that's cool." Then JS gave me the old Jurassic Park computer password warning.

Reply:
https://www.youtube.com/watch?v=rKX4KmtSBk8

Reply:
Classic! =)